sdk/config.js

```diff
@@ -12,9 +12,16 @@ const getLogEndpoint = function getLogEndpoint() {
   return process.env.BINARIS_LOG_ENDPOINT || 'log.binaris.com';
 };
 
+const forceRealm = function forceRealm(realm) {
+  process.env.BINARIS_DEPLOY_ENDPOINT = process.env.BINARIS_DEPLOY_ENDPOINT || `api-${realm}.binaris.com`;
```

Comment:
it's basically made for sandbox, which is in `binaris.com`, so I think we should leave it like this for now and now mess with it.

I can make the change refactor using `setDefaultEndpoints` you suggested but I like the functionality won't change.  
Because I'm doing
```
process.env.BINARIS_DEPLOY_ENDPOINT = process.env.BINARIS_DEPLOY_ENDPOINT || `api-${realm}.binaris.com`;
```
The function will now effect anything if the environment var is already there.